pkg/ctl/get/nodegroup_test.go

```diff
@@ -15,12 +17,41 @@ var _ = Describe("get", func() {
 		})
 
 		It("setting --name and argument at the same time", func() {
-			cmd := newMockCmd("nodegroup", "ng", "--cluster", "dummy", "--name", "ng")
```

Comment:
Any reason why the `--cluster` argument was removed from this test?

Reply:
This one is a bit of a interesting one. I think it was testing the wrong thing. However, with the introduction of the new `--config-file` option the error message is not correctly returning that `--name and nameArg can't be together, where as previously, it was doing the wrong thing with the `--cluster` param. Very weird. :D

Reply:
Uh, I just understood what's happening here and will provide a fix in an upcoming PR.